browser-test/src/admin_predicates.test.ts

```diff
@@ -371,12 +387,15 @@ describe('create and edit predicates', () => {
     await applicantQuestions.clickNext()
 
     // Earlier than 2021-01-01 is allowed
-    // TODO(#3859): 2021-01-01 evaluates as earlier, but it shouldn't.
-    await applicantQuestions.answerDateQuestion('2021-01-02')
+    await applicantQuestions.answerDateQuestion('2021-01-01')
     await applicantQuestions.clickNext()
     await applicantQuestions.expectReviewPage()
     await page.goBack()
-    await applicantQuestions.answerDateQuestion('2020-12-31')
+    await applicantQuestions.answerDateQuestion('2020-01-01')
```

Comment:
Suggest using the first date that is valid (2020-12-31). As is it could be almost a year off and we wouldn't detect it.